worksheet_A/TerminalHacking.cpp

```diff
@@ -40,7 +54,45 @@ int main()
 	}
 
 	// TODO: implement the rest of the game
+	int Likeness = 0;
+	int IncorrectGuesses = 0;
+	bool inlist;
+	std::string guess;
```

Comment:
Do all of these need to be declared outside the loop? Generally better to keep variable declarations close to the point of use.